### PR TITLE
Fb feature connect api

### PR DESCRIPTION
**Pull request by franblanco95**
https://user-images.githubusercontent.com/83734276/153238544-a83781da-caa2-4f51-b4c3-18046f760e84.mov

### CODE REVIEW

File: assets/colors/colors.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ const colors = {
     buttonbkg: '#FBD24D',
     input: '#B7C0C6',
     inputoff: '#FBD24D',
+    positive: '#008000',
```
Comment (tomasdematos) [inline assets/colors/colors.tsx:10]: [Minor] the hex name of #000000 is black,

**Review by tomasdematos — APPROVED** [review]

File: store/actions/crypto.actions.tsx [hunk 2]
```diff
@@ -20,31 +20,43 @@ export const readData = () => {
...
             .catch(err => { console.error(err) })
-
     }
 }
 
 export const addCripto = (textInput: string) => {
     return async (dispatch: Dispatch<Action>, getState: () => RootState) => {
+
         const { cryptoList } = getState().cripto
+        const response = await fetch(`https://data.messari.io/api/v1/assets/${textInput.toLowerCase()}/metrics?fields=id,symbol,name,market_data/price_usd,market_data/percent_change_usd_last_24_hours`)
```
Comment (ale1313) [inline store/actions/crypto.actions.tsx]: [CRITICAL] Host url should never be hardcoded in the app. Use environment variables instead

File: store/actions/crypto.actions.tsx [hunk 3]
```diff
@@ -20,31 +20,43 @@ export const readData = () => {
...
-                (textInput === name || textInput === initials)
-            )
-            return !addedCrypto && (name === textInput || initials === textInput)
-        })
-        if (newCrypto) {
-            dispatch({
-                type: ADD_CRYPTO,
-                payload: newCrypto
-            })
+        if (!response.ok) {
```
Comment (ale1313) [inline store/actions/crypto.actions.tsx]: [suggestion] This is not wrong, but best practices follow asserting conditionals. Please invert this condition so the positive affirmation comes first

File: store/actions/crypto.actions.tsx [hunk 4]
```diff
@@ -20,31 +20,43 @@ export const readData = () => {
...
+
+            let message = "Hubo un problema"
```
Comment (ale1313) [inline store/actions/crypto.actions.tsx]: [minor] Do not mix english and spanish, please use english whenever possible

File: store/actions/crypto.actions.tsx [hunk 5]
```diff
@@ -20,31 +20,43 @@ export const readData = () => {
...
+            const errorResponse = await response.json()
+            const errorID = errorResponse.status.error_message
+            if (errorID === "Not Found") message = "This cryptocurrency doesn't exist"
+            Alert.alert(message)
```
Comment (ale1313) [inline store/actions/crypto.actions.tsx]: [suggestion] There should be actions after this message pops up, e.g. retry the request, cancel, etc

File: store/actions/crypto.actions.tsx (hunk 2 above)
Comment (ale1313) [reply store/actions/crypto.actions.tsx]: [minor] Api requests should be wrapped in try-catch blocks

File: store/actions/crypto.actions.tsx [hunk 6]
```diff
@@ -20,31 +20,43 @@ export const readData = () => {
...
-            } catch (e) {
-                console.error(e)
+        } else {
+
+            const result = await response.json()
+
+            const addedCrypto: Crypto | undefined = cryptoList.find(({ name, symbol }) =>
+                (textInput.toLowerCase() === name.toLowerCase() || textInput.toLowerCase() === symbol.toLowerCase())
+            )
+            if (!addedCrypto) {
```
Comment (ale1313) [inline store/actions/crypto.actions.tsx]: [suggestion] Same as above, please try to work with affirmative conditionals whenever possible

**Review by ale1313 — APPROVED** [review]

**Pull request merged**